### PR TITLE
Set files for review

### PR DESCRIPTION
**Pull request by JohnFTitor**
- Didn't change any file. Just add linters to check if the project meets the requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,5 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# FCC Test Suite Template
```
Comment (JohnFTitor) [inline README.md:1]: - [ ] It seems to be your project is based on a template. While it's good to give recognition to the original author, keep in mind that you still need to document your code professionalism, so other people can understand what your project is about. You can always give recognition through an Acknowledgment section.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(27, 27, 27);
+}
+
+#projects h1{
+    font-size: 2.5rem;
+    color: rgb(27, 121, 63);
+}
```
Comment (JohnFTitor) [inline css/style.css:44]: - [ ] As a suggestion, try not to style too much on an id, as it is less flexible for larger projects when you might need to apply similar attributes. Check for example there are four attributes with the same values between these two

File: index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav id="navbar">
+      <a href="#welcome-section"> Home </a>
+      <a href="#projects"> Projects </a>
```
Comment (JohnFTitor) [inline index.html:16]: - [ ] A good practice is to group `<a>` tags within `<ul>` tags, each hyperlink being a `<li>` element

File: index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  </section>
+
+  <section id="projects" class="container">
+    <h1>Our Projects</h1>
+    <div id="all-tiles" class="container">
+      <div class="project-tile">
+        <a href="https://qwibbler.github.io/DanteZaman/" target="_blank" text-decoration: none;>
+          <img class="tile-img" src="img/tp.png"
+            alt="Tribute Page">
+          <h2>Tribute Page</h2>
```
Comment (JohnFTitor) [inline index.html:30]: - [ ] Remember the headings hierarchy whenever you structure your HTML as this provides better accessibility in the way a browser understands your content. Even if they are in different sections, if possible, try to divide your content importance and thus make use of the headings

**Review by JohnFTitor — COMMENTED** [review]
# Greetings @JohnFTitor 👋🏻 

Your project is doing well so far 💯 💯. Good job during your trials.  
There are some things that still need some work, but you're almost there! 💪🏻.



Let's start by highlighting what's good.

- ✅ Files structure is organized in folders
- ✅ The project meets FreeCodeCamp requirements 
- ✅ The project is similar to the template provided 

## Now let's check some suggestions to make this project even better 

 ### Linters 

Your project doesn't meet linters requirements. I suggest you install them locally to check them before you push the files. Remember linters helps you build a project with better programming practices.

### Comments on the files

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 

### Hello @JohnFTitor  ✋🏼 
#### Your project is great. I approve of it. There is nothing else to say other than... it's time to merge it and delete the branch. You can submit this project on your dashboard.

### 💯  Nice to have 💯
- You phrased the comments politely, as suggestions instead of orders.
- Comments are clear and easy to understand.
- Your comments include an explanation, a "why", for each requested change.
- You know how to write inline Github comments.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ]  I would encourage the use of a couple of emojis and/or gifs in all of your reviews. They make the code review more human-friendly 👨🏼  👩🏼  not  🤖 👽 

### Cheers and Happy coding! 💻 👏
aproved

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**